app/src/main/java/org/sil/gatherwords/MainActivity.java

```diff
@@ -17,6 +23,27 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        /* Begin code by Joseph Jinn */
```

Comment:
In most cases, we do not label our code like this (includes the other instances).  If it becomes necessary to determine the author, we can use git history.